monthly-exams/exam-1/src/App.css

```diff
@@ -0,0 +1,27 @@
+.container {
```

Comment:
Your container does not have enough CSS so it does not stretch the entire screen, in this case, you want 
```
width: 100vw;
height: 100vh;
```
to force the `.container` stretch the whole screen